WcaOnRails/spec/iframe_mail_interceptor_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class TestMailer < ApplicationMailer
+  def send_mail(body)
+    mail(
+      to: "receiver@example.com",
+      subject: "RSpec email",
+      body: body,
+    )
+  end
+end
+
+RSpec.describe IframeMailInterceptor do
+  let(:iframe_email) { TestMailer.send_mail('<p><iframe src="https://www.worldcubeassociation.org"></iframe></p>') }
+  let(:yt_iframe_email) { TestMailer.send_mail('<p><iframe src="https://www.youtube.com/embed/VIDEO_ID"></iframe></p>') }
+  let(:gmaps_iframe_email) { TestMailer.send_mail('<p><iframe src="https://www.google.com/maps/embed/v1/place?key=API_KEY&q=USA"></iframe></p>') }
```

Comment:
Hmm... are the let's really doing anything for us here? Since they're only used in each individual test, why not just put them in the tests?

Reply:
Yeah, they don't add to readability or clearness so I'll remove them =)